code/modules/industrial_lift/tram_override_objects.dm

```diff
@@ -0,0 +1,44 @@
+/**
+ * the tram has a few objects mapped onto it at roundstart, by default many of those objects have unwanted properties
+ * for example grilles and windows have the atmos_sensitive element applied to them, which makes them register to
+ * themselves moving to re register signals onto the turf via connect_loc. this is bad and dumb since it makes the tram
+ * more expensive to move.
+ *
+ * if you map something on to the tram, make SURE if possible that it doesnt have anythign reacting to its own movement
+ * it will make the tram more expensive to move and we dont want that because we dont want to return to the days where
+ * the tram took a third of the tick per movement when its just carrying its default mapped in objects
+ */
+
+/obj/structure/grille/tram
```

Comment:
these defines aren't needed

Reply:
i know i just think its clearer

Reply:
maybe its just me but defining type and proc in the same line is possible via behavior that has fucked me over in the past when i mispell a typepath and so it compiles an important proc to that newly defined type